data_dictionary/tables.md

```diff
@@ -59,20 +105,43 @@ Holds information relevant to consultation requests for patients.
 
 | Name | Type | Description |
 |---| --- |---|
-| consultationRequestId | bigint | Unique identifier in EMAP for this **ConsultationRequest** record. |
-| consultationTypeId | [ConsultationType](#ConsultationType) | Identifier for the [ConsultationType](#ConsultationType) associated with this record. |
-| hospitalVisitId | [HospitalVisit](#HospitalVisit) | Identifier for the [HospitalVisit](#HospitalVisit) associated with this record. |
-| internalId | bigint | Identifier used in source system for this **ConsultationRequest**. |
-| closedDueToDischarge | boolean | Predicate determining whether this **ConsultationRequest** was closed on discharge. |
+| consultation_request_id | bigint | Unique identifier in EMAP for this **ConsultationRequest** record. |
+| consultation_type_id | [ConsultationType](#ConsultationType) | Identifier for the [ConsultationType](#ConsultationType) associated with this record. |
+| hospital_visit_id | [HospitalVisit](#HospitalVisit) | Identifier for the [HospitalVisit](#HospitalVisit) associated with this record. |
+| internal_id | bigint | Identifier used in source system for this **ConsultationRequest**. |
+| closed_due_to_discharge | boolean | Predicate determining whether this **ConsultationRequest** was closed on discharge. |
 | comments | varchar(255) | Notes added to the **ConsultationRequest** which are not tied to a Question. |
-| statusChangeTime | timestamp without timezone | Date and time at which this **ConsultationRequest** was last updated. |
-| scheduledDatetime | timestamp without timezone | Date and time at which this **ConsultationRequest** was scheduled. |
+| status_change_time | timestamp without timezone | Date and time at which this **ConsultationRequest** was last updated. |

```

Comment:
should that not be status_change_datetime? or is that not come through yet on develop?

Reply:
It's not been merged into master and I anticipate this PR going straight to main? As to reflect the current state of  the live DB

> NOTE: the changed tables.md was generated from the latest release tag

Reply:
doesn't really make a difference at the moment as I think we'll be releasing soon, but this should reflect what is currently in `master` (and released to the users!), not what's in develop